### PR TITLE
YNGJ-1058: Create Aether Observatory

### PR DESCRIPTION
**Pull request by dutchess**
Extracting `AetherObservatory` from `Talkbox`.  Internal PubSub system base upon `ActiveSupport::Notifications`

### CODE REVIEW

File: .github/workflows/aether_observatory.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+on:
+  push:
+
+jobs:
+  ruby:
+    uses: powerhome/github-actions-workflows/.github/workflows/ruby-gem.yml@main
+    with:
+      package: ${{ github.workflow }}
+      workdir: "packages/${{ github.workflow }}"
+      ruby: '["3.0","3.1","3.2","3.3"]'
```
Comment (xjunior) [inline .github/workflows/aether_observatory.yml:12]: Most of our libraries are testing things in all these rubies, but in reality all we need is 3.0 and 3.3.

File: packages/aether_observatory/aether_observatory.gemspec [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  spec.files = Dir.chdir(__dir__) do
+    `git ls-files -z`.split("\x0").reject do |f|
+      (f == __FILE__) || f.match(%r{\A(?:(?:bin|test|spec|features)/|\.(?:git|travis|circleci)|appveyor)})
+    end
+  end
+  spec.require_paths = ["lib"]
+
+  spec.add_dependency "activemodel", ">= 6.0.6.1"
+  spec.add_dependency "activesupport", ">= 6.0.6.1"
+  spec.add_development_dependency "appraisal", "~> 2.5.0"
```
[the remark this answers is not in the document]
Comment (dutchess) [reply packages/aether_observatory/aether_observatory.gemspec:33]: Thanks y'all.  I meant to circle back to this, but forgot.

**Comment by dutchess** [comment]
@that-jill and @xjunior I think I've got Appraisal set up correctly.  Should it be testing all packages on all pushes?

I tagged a version as v0.0.1pre4 that finally made it to rubygems.

**Review by rizenine — DISMISSED** [review]

File: packages/aether_observatory/aether_observatory.gemspec (hunk 2 above)
[the remark this answers is not in the document]
Comment (dutchess) [reply packages/aether_observatory/aether_observatory.gemspec:33]: Changed the Appraisal

**Review by xjunior — APPROVED** [review]

**Pull request merged**